### PR TITLE
CR feedback

### PR DESCRIPTION
**Pull request by smk2007**
User/sheilk/windowsai merge cr feedback

### CODE REVIEW

File: cmake/CMakeLists.txt [hunk 1]
```diff
@@ -84,7 +84,6 @@ option(onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS "Dump node input shapes and output
 option(onnxruntime_USE_DML "Build with DirectML support" OFF)
 option(onnxruntime_USE_WINML "Build with WinML support" OFF)
 option(onnxruntime_USE_ACL "Build with ACL support" OFF)
-option(onnxruntime_USE_TELEMETRY "Build with Telemetry" OFF)
```
Comment (ryanlai2) [inline cmake/CMakeLists.txt]: Why remove this?

File: include/onnxruntime/core/session/onnxruntime_c_api.h [hunk 2]
```diff
@@ -650,6 +650,66 @@ struct OrtApi {
...
+  // End of Version 1 - DO NOT MODIFY ABOVE (see above text for more information)
+
+  // Version 2 - In development, feel free to add/remove/rearrange here
+
+  /**
+    * GetDenotationFromTypeInfo
+	 * This api augments OrtTypeInfo to return denotations on the type.
+	 * This is used by WinML to determine if an input/output is intended to be an Image or a Tensor.
+    */
+  OrtStatus*(ORT_API_CALL* GetDenotationFromTypeInfo)(_In_ const OrtTypeInfo*, _Out_ const char** const denotation, _Out_ size_t* len)NO_EXCEPTION;
```
Comment (fdwr) [inline include/onnxruntime/core/session/onnxruntime_c_api.h]: [nit] Add some spaces for readability before and after the function? Or is stupid clang-format messing things up?

`OrtStatus* (...)(...) NO_EXCEPTION;`

File: cmake/CMakeLists.txt (hunk 1 above)
Comment (emigsheil) [reply cmake/CMakeLists.txt]: Because it is duped a few lines below.

Comment (smk2007) [reply cmake/CMakeLists.txt]: Because it is duped a few lines below.

File: include/onnxruntime/core/session/onnxruntime_c_api.h (hunk 2 above)
Comment (fdwr) [reply include/onnxruntime/core/session/onnxruntime_c_api.h]: Ah, seems you just cut&paste this from elsewhere. Non-blocking minor.

File: docs/Privacy.md [hunk 3]
```diff
@@ -14,5 +14,8 @@ Currently telemetry is only implemented for Windows builds, but may be expanded
 #### Technical Details
 The Windows provider uses the [TraceLogging](https://docs.microsoft.com/en-us/windows/win32/tracelogging/trace-logging-about) API for its implementation.
 
-Windows ML and onnxruntime C APIs allow telemetry collection to be turned on/off (see API pages for details). In all Official builds, the Windows ML API enables telemetry collection.
+Windows ML and onnxruntime C APIs allow telemetry collection to be turned on/off (see API pages for details).
+In all Official builds:
+- Telemetry collection is *disabled* for calls to the ORT API.
+- Telemetry collection is *enabled* for calls to the Windows ML API.
```
Comment (tiagoshibata) [inline docs/Privacy.md]: This looks weird:

image

The C API link should follow `see API pages for details`, now it's in the end of the `In all Official builds` list

File: include/onnxruntime/core/session/onnxruntime_c_api.h (hunk 2 above)
Comment (smk2007) [reply include/onnxruntime/core/session/onnxruntime_c_api.h]: same formatting as other places in this file!

---
In reply to: [374407768](https://github.com/microsoft/onnxruntime/pull/2970#discussion_r374407768) [](ancestors = 374407768)

Comment (smk2007) [reply include/onnxruntime/core/session/onnxruntime_c_api.h]: yep

---
In reply to: [374408058](https://github.com/microsoft/onnxruntime/pull/2970#discussion_r374408058) [](ancestors = 374408058)

**Comment by smk2007** [comment]
YW!

---
In reply to: [352644894](https://github.com/microsoft/onnxruntime/pull/2970#issuecomment-352644894) [](ancestors = 352644894)

File: docs/Privacy.md (hunk 3 above)
Comment (emigsheil) [reply docs/Privacy.md]: moved around.

Comment (emigsheil) [reply docs/Privacy.md]: moved around

Comment (smk2007) [reply docs/Privacy.md]: moved around.

File: cmake/precompiled_header.cmake [hunk 4]
```diff
@@ -1,3 +1,6 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (ryanlai2) [inline cmake/precompiled_header.cmake]: >All rights reserved. [](start = 39, length = 20)

Why keep "All rights reserved" here and remove it for the source files?

Comment (smk2007) [reply cmake/precompiled_header.cmake]: To make it match everywhere else.

---
In reply to: [374411856](https://github.com/microsoft/onnxruntime/pull/2970#discussion_r374411856) [](ancestors = 374411856)

Comment (ryanlai2) [reply cmake/precompiled_header.cmake]: Same thing for the other .cmake files and also in CMakeLists.txt?

---
In reply to: [374412145](https://github.com/microsoft/onnxruntime/pull/2970#discussion_r374412145) [](ancestors = 374412145,374411856)

Comment (smk2007) [reply cmake/precompiled_header.cmake]: hmm... seems like All rights reserved is more common in the code base... switching to that.

---
In reply to: [374412543](https://github.com/microsoft/onnxruntime/pull/2970#discussion_r374412543) [](ancestors = 374412543,374412145,374411856)

File: include/onnxruntime/core/providers/winml/winml_provider_factory.h [hunk 5]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (zhangxiang1993) [inline include/onnxruntime/core/providers/winml/winml_provider_factory.h]: Revert?

File: winml/adapter/winml_adapter_dml.cpp [hunk 6]
```diff
@@ -1,5 +1,6 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (zhangxiang1993) [inline winml/adapter/winml_adapter_dml.cpp]: revert?

File: winml/adapter/winml_adapter_environment.cpp [hunk 7]
```diff
@@ -1,5 +1,6 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (zhangxiang1993) [inline winml/adapter/winml_adapter_environment.cpp]: revert?

File: winml/adapter/winml_adapter_execution_provider.cpp [hunk 8]
```diff
@@ -1,5 +1,6 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (zhangxiang1993) [inline winml/adapter/winml_adapter_execution_provider.cpp]: same.

File: cmake/precompiled_header.cmake [hunk 9]
```diff
@@ -1,3 +1,6 @@
+# Copyright (c) Microsoft Corporation.
```
Comment (zhangxiang1993) [inline cmake/precompiled_header.cmake]: All rights reserved.

File: tools/ci_build/github/azure-pipelines/win-gpu-ci-pipeline.yml [hunk 10]
```diff
@@ -42,8 +42,7 @@ jobs:
     displayName: 'Generate cmake config'
     inputs:
       scriptPath: '$(Build.SourcesDirectory)\tools\ci_build\build.py'
-      arguments: '--config $(BuildConfig) --build_dir $(Build.BinariesDirectory) --skip_submodule_sync --build_shared_lib --update --cmake_generator "Visual Studio 16 2019" --msvc_toolset 14.16 --build_wheel --use_featurizers 
-      --use_dnnl --build_shared_lib --enable_onnx_tests --use_dml --use_winml --use_cuda --cuda_version=10.0 --cuda_home="C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0" --cudnn_home="C:\local\cudnn-10.0-windows10-x64-v7.6.5.32\cuda" --cmake_extra_defines CMAKE_SYSTEM_VERSION=10.0.18362.0'
+      arguments: '--config $(BuildConfig) --build_dir $(Build.BinariesDirectory) --skip_submodule_sync --build_shared_lib --update --cmake_generator "Visual Studio 16 2019" --msvc_toolset 14.16 --build_wheel --use_featurizers --use_dnnl --build_shared_lib --enable_onnx_tests --use_dml --use_cuda --cuda_version=10.0 --cuda_home="C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0" --cudnn_home="C:\local\cudnn-10.0-windows10-x64-v7.6.5.32\cuda" --cmake_extra_defines CMAKE_SYSTEM_VERSION=10.0.18362.0'
```
Comment (zhangxiang1993) [inline tools/ci_build/github/azure-pipelines/win-gpu-ci-pipeline.yml]: how about two lines?

**Comment by zhangxiang1993** [comment]
>      python $(Build.SourcesDirectory)\tools\ci_build\build.py --config $(BuildConfig) --build_dir $(Build.BinariesDirectory) --skip_submodule_sync --build_shared_lib --test --cmake_generator "Visual Studio 16 2019" --msvc_toolset 14.16 --build_wheel --use_featurizers --use_dnnl --build_shared_lib --enable_onnx_tests --use_dml --use_cuda --cuda_version=10.0 --cuda_home="C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0" --cudnn_home="C:\local\cudnn-10.0-windows10-x64-v7.6.5.32\cuda" --cmake_extra_defines CMAKE_SYSTEM_VERSION=10.0.18362.0

nit: two lines

---
Refers to: tools/ci_build/github/azure-pipelines/win-gpu-ci-pipeline.yml:119 in 784cefa. [](commit_id = 784cefa61df3babb977dc841badd88ae9161bdb5, deletion_comment = False)

**Pull request merged**